xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -17,17 +17,22 @@
  *  <http://www.gnu.org/licenses/>.
  *
  */
+#include "system_gl.h"
```

Comment:
please don't use this crappy header. make it simple and obvious what you want to include.